src/engine/__tests__/parser/index.spec.js

```diff
@@ -168,12 +183,13 @@ tape('PARSER / if user.account is in segment all then split 100:on', assert => {
     partitions: [{
       treatment: 'on',
       size: 100
-    }]
+    }],
+    label: 'in segment all'
```

Comment:
but this is not a label constant, 

here you can see what are a label constant and which not 
https://docs.google.com/document/d/1D1Bz5_VTbfns6woNwyImPUQ7f0ajr3B7NPw7jVIKYug/edit